package.json

```diff
@@ -33,6 +33,7 @@
     "build:dapp": "yarn workspace @ubiquity/dapp run build",
     "dev:dapp": "yarn workspace @ubiquity/dapp run start",
     "test:all": "yarn workspace @ubiquity/contracts run test:unit",
+    "coverage": "yarn workspace @ubiquity/contracts run test:coverage",
```

Comment:
There is not such command `test:coverage` here https://github.com/eugenioclrc/ubiquity-dollar/blob/635-remove-unnecessary-files-from-coverage/packages/contracts/package.json. Where is it located?